### DISCUSSION TITLE
AUC ratio calculation in PK-Sim after multiple doses

### DISCUSSION THREAD

**Discussion by teutonicod** [Open-Systems-Pharmacology/Forum#910] [Questions & Problems]
Hi, 

I am using PK-Sim to perform some DDI calculation. I notice that sometimes there is a difference in the AUCR calculated by PK-Sim and the resulting AUCR which can be obtained between the simulation with the inhibitor and a control simulation without it. In the last case where I notice this, it was in a repeated dose simulation, so I will focus on this case. 
I simulated a repeated dose of my victim drug with and without perpetrator, and I would like to know the AUCR at steady state. I computed the ratio of the AUC previous to the last administration (the one provided by PK-Sim) with and without inhibitor. Then I let PK-Sim calculate the ration in the simulation containing the perpetrator.

If I correctly understood the tooltips, with repeated doses, PK-Sim computes the ratio using the AUC of the last administration, and not the previous-to-last as in my manual calculation. Is this the case? Since this could eventually explain the difference.
Is there any other difference in a control simulation without perpetrator and the "control" generated by PK-Sim when asking to get the AUCR?

Hope I was clear enough with my question!

Donato

**Comment by PL-FE** [comment]
已经收到您的来信，我会尽快回复。
